src/structures/role.ts

```diff
@@ -129,6 +129,7 @@ export async function createDiscordenoRole(
 
   const role: DiscordenoRole = Object.create(baseRole, {
     ...props,
+    permissions: createNewProp(snowflakeToBigint(rest.permissions)),
```

Comment:
```suggestion
    permissions: createNewProp(BigInt(rest.permissions)),
```

Reply:
this is perms not a snowflake although result is same, using it here is confusing